src/main/java/com/frcteam3636/frc2024/Robot.kt

```diff
@@ -126,18 +143,41 @@ object Robot : LoggedRobot() {
 
 
     private fun configureBindings() {
-       Drivetrain.defaultCommand = Drivetrain.driveWithJoysticks(
-           joystickLeft, joystickRight
-       )
 
-//       controller.leftBumper().whileTrue(Intake.outtakeComand())
+        Drivetrain.defaultCommand = Drivetrain.driveWithJoysticks(
+            joystickLeft,joystickRight
+        )
+
+        JoystickButton(joystickLeft, 2).whileTrue(
+            Drivetrain.driveWithJoystickPointingTowards(joystickLeft, SPEAKER_POSE.toTranslation2d())
+        ).onFalse(
+            InstantCommand({
+                Drivetrain.defaultCommand = Drivetrain.driveWithJoysticks(
+                    joystickLeft, joystickRight
+                )
+            })
+        )
```

Comment:
This probably isn't necessary because the default command never changes. And when the whileTrue command ends it will cancel the driveWithJoystickPointingTowards anyways